dataflux_pytorch/dataflux_mapstyle_dataset.py

```diff
@@ -90,7 +93,7 @@ def __init__(
         project_name,
         bucket_name,
         config=Config(),
-        data_format_fn=lambda data: data,
+        data_format_fn=None,
```

Comment:
Why is this necessary?

Reply:
Multiprocessing in pytorch dataloader cannot pickle lambda functions. 
Getting error: 
"_pickle.PicklingError: Can't pickle <function DataFluxIterableDataset.<lambda> at 0x10a623550>: attribute lookup DataFluxIterableDataset.<lambda> on dataflux_pytorch.dataflux_iterable_dataset failed"